docs/Comparison.md

```diff
@@ -2,23 +2,24 @@
+> Chart by [Carl George](https://www.linkedin.com/in/carlwgeorge/)
+
+![relationship_chart](/images/Comparison-relationship_chart.png)
+
+The chart above illustrates the relationship between Fedora, CentOS Stream, and RHEL.<br>
+AlmaLinux utilizes package sources from both RHEL and CentOS Stream to build its distribution. In contrast, Oracle Linux and Rocky Linux rely exclusively on package sources from RHEL.
+
+|Benchmarking against RHEL       |AlmaLinux                                 |Oracle Linux                        |Rocky Linux                                                           |CentOS Stream                  |RHEL                           |
+|--------------------------------|------------------------------------------|------------------------------------|----------------------------------------------------------------------|-------------------------------|-------------------------------|
+|First release                   |Since March 2021                          |Since 2006                          |Since June 2021                                                       |Since 2019                     |March 2002                     |
+|RHEL compatibility              |minor version compatible                  |minor version compatible            |minor version compatible                                              |major version compatible       |n/a                            |
+|Last minor version release delay|[5 days][alma9.4]                         |[6 days][oracle9.4]                 |[8 days][rocky9.4]                                                    |N/A                            |n/a                            |
+|Errata                          |Yes                                       |Yes                                 |Yes                                                                   |No                             |yes                            |
+|Lifecycle                       |10 Years                                  |10 Years                            |10 Years                                                              |5.5 Years                      |10-14 years                    |
+|Commercial support              |3rd party                                 |Oracle, 3rd parties                 |3rd parties                                                           |3rd Party                      |1st party (Red Hat)            |
+|Livepatching service            |KernelCare                                |Oracle Ksplice, KernelCare Available|KernelCare Available                                                  |Not available                  |kpatch                         |
+|Architectures                   |aarch64, ppc64le, s390x                   |aarch64                             |aarch64, ppc64le (versions ≥ 9), s390x (versions ≥ 9)                 |aarch64, ppc64le               |aarch64, ppc64le, s390x, x86_64|
```

Comment:
Alma, Rocky, and CentOS Stream are all available for the same four architectures as RHEL: aarch64, ppc64le, s390x, and x86_64.  Oracle is available for aarch64 and x86_64.

Reply:
TBH I just took what was on the previous table and used that as my source of information, thanks for clarifying.